src/overlays/actors/ovl_En_Zog/z_en_zog.c

```diff
@@ -994,7 +994,7 @@ void EnZog_PostLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3s* rot,
     }
 }
 
-void func_80B95598(PlayState* play, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx, Gfx** gfx) {
+void EnZog_PostLimbDraw2(PlayState* play, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx, Gfx** gfx) {
```

Comment:
Same here. I would argue this should be `EnZow_PostLimbDraw` and the other `EnZow_PostLimbDrawOpa`